test/cpp/c10d/ProcessGroupNCCLTest.cpp

```diff
@@ -731,3 +878,16 @@ TEST_F(ProcessGroupNCCLTest, testBackendName) {
         std::string(c10d::NCCL_BACKEND_NAME));
   }
 }
+
+#ifdef IS_NCCL_EXP
+TEST_F(ProcessGroupNCCLTest, testSparseAllreduce) {
+  if (skipTest()) {
+    return;
+  }
+  {
+    TemporaryFile file;
+    testSparseAllreduce(file.path, rank_, size_);
+    // testSparseAllreduceLarge(file.path, rank_, size_);
```

Comment:
does the test work?

Reply:
yes, both work and operate on different sizes. Will uncomment it